frontend/test/metabase-smoketest/admin_setup.cy.spec.js

```diff
@@ -930,40 +929,38 @@ describe("smoketest > admin_setup", () => {
       cy.findByText("Sorry, you don’t have permission to see that.");
     });
 
-    it("should deactivate a user admin and subsequently user should be unable to login", () => {
-      // Admin deactiviates user
+    it("user should not be able to login after admin deactivated them", () => {
+      const FULL_NAME = normal.first_name + " " + normal.last_name;
 
       cy.signOut();
       cy.signInAsAdmin();
-      cy.visit("/admin/settings/setup");
 
+      cy.visit("/admin/settings/setup");
       cy.findByText("People").click();
-      cy.icon("ellipsis")
-        .eq(-2)
-        .click();
-      cy.findByText("Deactivate user").click();
 
-      cy.findByText("Robert Tableton won't be able to log in anymore.");
+      openEllipsisMenuForUser(FULL_NAME);
+      cy.findByText("Deactivate user").click();
 
-      cy.findByText("Deactivate").click();
-
-      cy.findByText(normal.first_name + " " + normal.last_name).should(
-        "not.exist",
-      );
-      cy.findByText(new_user.first_name + " " + new_user.last_name);
+      cy.findByText(`${FULL_NAME} won't be able to log in anymore.`);
+      cy.button("Deactivate").click();
+      cy.findByText(FULL_NAME).should("not.exist");
```

Comment:
Maybe these could be extracted into a function named `deactivateUser`..

Then this test could become really easy to grasp:
sign out, sign in, deactivate user, try to sign, check that passwords don't match.

Reply:
+1 here. The test is about users not being able no log in once their account is deactivated, not about the deactivation itself. So I agree it can improve readability

Anyway, this PR is about getting rid of TLDs in tests, so we can address that later as I think

Reply:
I hear you both.
These smoke tests are in such a bad shape, though. We can dedicate one whole PR just for fixing them.
Most of them are even non-sensical.

What I'm trying to say is:
1. This is way out of scope of this PR
2. There will be much bigger changes and improvements are needed to make this more readable, but that comes after FULLY re-thinking what smoke tests are even doing in the first place.